src/types/onyx/Policy.ts

```diff
@@ -5,18 +5,34 @@ import type * as OnyxCommon from './OnyxCommon';
 type Unit = 'mi' | 'km';
 
 type Rate = {
-    name: string;
-    rate: number;
-    currency: string;
+    name?: string;
+    rate?: number;
+    currency?: string;
+    customUnitRateID?: string;
+    errors?: OnyxCommon.Errors;
+    pendingAction?: string;
 };
 
-type CustomUnit = {
+type Attributes = {
+    unit: Unit;
+};
+
+type NewCustomUnit = {
```

Comment:
We should not need separate types for `CustomUnit` and `NewCustomUnit`.

Furthermore, It's pretty sus that all the fields of both would be optional.